tests/conftest.py

```diff
@@ -292,3 +292,8 @@ def mechanical_pool():
 
     assert f"Mechanical pool with {instances_count} active instances" in str(pool)
     pool.exit(block=True)
+
+
+def pytest_addoption(parser):
+    # parser.addoption("--debugging", action="store_true")
```

Comment:
```suggestion
```
Remove it or uncomment it, as you prefer. I'd avoid having commented out code.

Reply:
@RobPasMue I have the same preference, but I do intend to revisit this soon